simpleUpload/index.html

```diff
@@ -1,96 +1,35 @@
-    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <meta name="description" content="Mapillary to Wikimedia Commons uploader">
-    <meta name="author" content="Lokal_Profil">
-    <link rel="icon" href="../../favicon.ico">
-    <title>Mapillary2Commons</title>
+	<head>
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="Mapillary to Wikimedia Commons uploader" />
+		<title>Mapillary 2 Commons</title>
+		<link rel="stylesheet" href="css/style.css" />
+	</head>
+	<body>
+		<div class="notification" id="notification"></div>
+		<header>
+			<div class="container">
+				<h1>Mapillary 2 Commons</h1>
+				<ul>
+					<li>You need a Wikimedia Commons account and to have <a href="https://tools.wmflabs.org/magnustools/oauth_uploader.php?action=authorize">authorized OAuth Uploader</a> for this tool to work.</li>
+					<li>Don't forget to go to the Commons image afterwards and add categories.</li>
+				</ul>
+				<input id="mapillary-input" type="text" placeholder="https://www.mapillary.com/map/im/2z5MCCrTgC1T29Gm8XJrCg" />
+				<br/>
+				<button id="mapillary-submit" class="btn white">Lookup</button>
+			</div>
+		</header>
+		<main id="main" class="container">
+			<img id="image" alt="the image to import" src="" />
+			<input id="location-input" type="text" placeholder="Describe location" />
 
-    <!-- Bootstrap -->
-    <link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet">
+			<p>File to be named: <i id=filename-label></i></p>
+			<a id="upload" href="" class="btn">Upload</a>
+		</main>
 
-    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
-    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
-    <!--[if lt IE 9]>
-      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
-      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
-    <![endif]-->
-
-    <!-- my stuff -->
-    <link rel='stylesheet' type='text/css' href='stylesheet.css'/>
-  </head>
-  <body>
-    <noscript>
-            <div class="text-center text-danger">
-                This web page requires JavaScript. Sorry?
-            </div>
-        </noscript>
-        <div class="container">
-            <div class="page-header">
-                <h1>Mapillary 2 Commons</h1>
-            </div>
-            <p class="lead">
-                Ever found a beautiful image on <a href="https://www.mapillary.com">Mapillary</a>
-                and thought <i>That would look great on <a href="https://commons.wikimedia.org">Wikimedia Commons</a></i>?
-            </p>
-            <p> Simply enter the image key or url below and click <i>Check!</i>.
```

Comment:
No, not at all (in both cases), I started from scratch so I just didn't  think about it.